department-of-veterans-affairs/va.gov-cms
# VACMS-2525: Menu toolbar improvements.

## Description

See #2525. 

## Testing done

Manual. 

# #2525 

### Before 
![Disable_Node_Menu_Item_Settings___VA_gov_CMS](https://user-images.githubusercontent.com/643678/90948823-a1009100-e410-11ea-838b-f35dc54b19d1.jpg)

### After
![kevin_walsh___VA_gov_CMS](https://user-images.githubusercontent.com/643678/90948834-b2e23400-e410-11ea-845e-918e34e47af5.jpg)


## QA steps

Review as any user on PR environment AND locally

- [ ] Review string that shows the VACMS version for accuracy
- [ ] Review new Sections icon and all colors for accessibility
- [ ] To test prod environment indicator settings, overwrite lines 22-24 of settings.lando.php on your local with lines 22-24 of settings.prod.php and clear cache. 

## Notes

* This uninstalls moderation_sidebar. It offers nothing right now, but gets in the way of floating the Environment indicator to the right. We could probably remove it altogether. 
* on any screens that contain contextual links (like any node) there's an "edit" toolbar tab that appears on the top right. But Contextual links are only visible to admins right now, so i think this is fine. 

## Potential improvements

* Remove the Parentheses around VACMS v0.0.167, especially on prod. 


# #2660 CTA blocks

![VAMC_guides_by_content_type___Veterans_Affairs](https://user-images.githubusercontent.com/643678/90949772-52a3c000-e419-11ea-8297-bd1443e30d5e.jpg)

## QA steps

As admin, add the following classes to the new menu item attributes field for the top level menu items of the CMS Help Center block: 
* `icon cms-help-center`
* `icon helpdesk`
* `icon bug`
* `icon training`

Go to any page with the url `/help/*`, such as  /help/support/release-notes, and see these in the right rail. They should match the icons that appear in the in the CMS Help Center toolbar menu.
